src/pymax/interfaces.py

```
@@ -255,3 +162,391 @@ async def start(self) -> None:
+                        retry_delay,
+                    )
+
+                    if retry_count < max_retries:
+                        message["retry_count"] = retry_count + 1
+                        await asyncio.sleep(retry_delay)
+                        await self._outgoing.put(message)
+                    else:
+                        self.logger.error(
+                            "Message failed after %d retries, dropping",
+                            max_retries,
+                        )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)